app/controllers/admin/project_logs_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Admin::ProjectLogsController < Admin::AdminController
+  def destroy
+    @log = ProjectLog.find(params[:id])
+    @step = ProjectStep.find(@log.project_step_id)
+    @loan = Loan.find(@step.project_id)
+
+    if @log.destroy
+      redirect_to admin_loan_path(@loan), notice: I18n.t(:notice_deleted)
```

Comment:
Jumping down to the correct step will take some more planning. I tested out `anchor: 'timeline'`, which works, except it takes user to the very first project step, which is the top of `#timeline`.  I'll add jumping down to the correct step to the backlog as a tweak and think about that.